src/data/security/level1.yml

```diff
@@ -0,0 +1,39 @@
+# Security level 1: Home Workstation
+---
+CONSOLE_SHUTDOWN:                 reboot
+CRACKLIB_DICT_PATH:               "/usr/lib/cracklib_dict"
+DISPLAYMANAGER_REMOTE_ACCESS:     'no'
+kernel.sysrq:                     '1'
+FAIL_DELAY:                       '1'
+GID_MAX:                          '60000'
+GID_MIN:                          '1000'
+AllowShutdown:                    All
+HIBERNATE_SYSTEM:                 active_console
+PASSWD_ENCRYPTION:                sha512
+PASSWD_USE_CRACKLIB:              'yes'
+PASS_MAX_DAYS:                    '99999'
+PASS_MIN_DAYS:                    '0'
+PASS_MIN_LEN:                     '5'
+PASS_WARN_AGE:                    '7'
+PERMISSION_SECURITY:              easy
+RUN_UPDATEDB_AS:                  nobody
+UID_MAX:                          '60000'
+UID_MIN:                          '1000'
+SYS_UID_MAX:                      '499'
+SYS_UID_MIN:                      '100'
+SYS_GID_MAX:                      '499'
+SYS_GID_MIN:                      '100'
+USERADD_CMD:                      "/usr/sbin/useradd.local"
+USERDEL_PRECMD:                   "/usr/sbin/userdel-pre.local"
+USERDEL_POSTCMD:                  "/usr/sbin/userdel-post.local"
+PASSWD_REMEMBER_HISTORY:          '0'
+SYSTOHC:                          'yes'
+SYSLOG_ON_NO_ERROR:               'yes'
+DISPLAYMANAGER_ROOT_LOGIN_REMOTE: 'no'
+DISPLAYMANAGER_XSERVER_TCP_PORT_6000_OPEN: 'no'
```

Comment:
To be honest I do not know original one, but if I imagine myself that I add new option, I will be very lost where I should place it, so any logic is good, but if it is not obvious one like alphabetic, then please document it.